scripts/setup-helper-functions.sh

```diff
@@ -160,9 +160,16 @@ function wget_and_untar {
 }
 
 function cmake_install {
+  if [ -n "$1" ]; then
+    DIR="$1"
+    shift
+  else
+    DIR=$(pwd)
+  fi
   local NAME=$(basename "$(pwd)")
   local BINARY_DIR=_build
   SUDO="${SUDO:-""}"
+  cd "${DIR}"
```

Comment:
I see that the installs are happening within each directory `/fmt/folly/fizz/`
You have to make this `pushd "$DIR"` and add a `popd` at the end.